jina/proto/jina.proto

```diff
@@ -360,3 +361,13 @@ service JinaRPC {
     rpc Call (stream RequestProto) returns (stream RequestProto) {
     }
 }
+
+/**
+ * jina gRPC service for DataRequests.
+ */
+service JinaDataRequestRPC {
+    // Pass in a Message, wrapping a DataRequest
+    rpc Call (MessageProto) returns (google.protobuf.Empty) {
```

Comment:
Would unary call be enough here and we don't need a stream? And what does `returns ..Empty` mean?

Reply:
Every gRPC call needs to be return something, there is nothing like None or void. Empty is kind of the equivalent, it means that there will be an empty protobuf struct sent back as a result (and needs to be returned on the server side as well).
We dont want that really, but if we use RPC semantics I think there is no way around this hack